openapi_tester/constants.py

```diff
@@ -10,6 +10,23 @@
     "integer": int.__name__,
     "number": f"{int.__name__} or {float.__name__}",
 }
+OPENAPI_TYPE_EXAMPLES = {
```

Comment:
i intentionally used typings - i thiknk its clearer and more precise. So this change I am against (both constants here.). 

The rest of the PR is good. So please revert this particular change and then lets merge this.

Reply:
Sure can do. Just one follow up question: what do you feel about changing `json.dumps(example_item)` to just `example_item`? 

The confusing thing (to me) as that we're wrapping `int` in `"int"`. If we said something like 

```
Expected type: int

Received: "1"
```

I think it would make it much less confusing, potentially

Reply:
Checked, and while it makes the `int` example clearer, it probably makes more complex structures more confusing, as it just replaces all the double quotes with single quotes.

````
{"key1": "str", "key2": "str", "key3": "str", "key4": ["int"]}

Reply:
Reverted to the old map

Reply:
If we straight up replace quotes, we end up with `{key1: str, key2: str, key3: str, key4: [int]}`. Think that's an improvement.